calicoctl/calicoctl.go

```diff
@@ -20,8 +20,9 @@ import (
 	"strings"
 
 	"github.com/docopt/docopt-go"
-	"github.com/projectcalico/calicoctl/calicoctl/commands"
 	log "github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calicoctl/calicoctl/commands"
```

Comment:
One more comment while I was looking at docs. Can you update the usage doc below for the appropriate sub commands to state "filename, directory or stdin"?